tests/test-cases/Group3-Docker-Compose/3-2-Docker-Compose-Voting-App.robot

```diff
@@ -1,15 +1,21 @@
 *** Settings ***
 Documentation  Test 3-2 - Docker Compose Voting App
 Resource  ../../resources/Util.robot
-#Suite Setup  Install VIC Appliance To Test Server  ${false}
-#Suite Teardown  Cleanup VIC Appliance On Test Server
+Suite Setup  Install VIC Appliance To Test Server  ${false}
+Suite Teardown  Cleanup VIC Appliance On Test Server
 
 *** Test Cases ***
 Compose Voting App
-    ${status}=  Get State Of Github Issue  2358
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 3-2-Docker-Compose-Voting-App.robot needs to be updated now that Issue #2358 has been resolved
-    Log  Issue \#2358 is blocking implementation  WARN
-    #Run  git clone https://github.com/docker/example-voting-app test
-    #${rc}  ${output}=  Run And Return Rc And Output  cd test; DOCKER_HOST=${vch-ip}:2375 docker-compose up
-    #Log  ${output}
-    #Should Be Equal As Integers  ${rc}  0
\ No newline at end of file
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} login --username=victest --password=vmware!123
+    Should Be Equal As Integers  ${rc}  0
+    ${out}=  Run Process  docker-compose  up  -d  shell=True  cwd=${CURDIR}/../../../demos/compose/voting-app   env:COMPOSE_HTTP_TIMEOUT=300  env:DOCKER_HOST=${vch-ip}:2375
+    Log  ${out.rc}
```

Comment:
Needs to be ${out.rc}, don't need all the log statements above that was just and example of how to use the return structure from run process